src/main/java/pl/ug/virtualofficebackend/domain/entity/DecorationLayoutEntity.java

```diff
@@ -0,0 +1,138 @@
+package pl.ug.virtualofficebackend.domain.entity;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(name = "decoration_layout", schema = "public", catalog = "postgres")
+public class DecorationLayoutEntity {
+    private int decorationId;
+    private int officeId;
+    private int x1Location;
+    private int y1Location;
+    private int x2Location;
+    private int y2Location;
+    private String rotation;
+    private DecorationEntity decorationByDecorationId;
+    private OfficeEntity officeByOfficeId;
+
+    @Id
+    @Column(name = "decoration_id")
+    public int getDecorationId() {
+        return decorationId;
+    }
+
+    public void setDecorationId(int decorationId) {
+        this.decorationId = decorationId;
+    }
+
+    @Basic
+    @Column(name = "office_id")
+    public int getOfficeId() {
+        return officeId;
+    }
+
+    public void setOfficeId(int officeId) {
+        this.officeId = officeId;
+    }
+
+    @Basic
+    @Column(name = "x1_location")
+    public int getX1Location() {
+        return x1Location;
+    }
+
+    public void setX1Location(int x1Location) {
+        this.x1Location = x1Location;
+    }
+
+    @Basic
+    @Column(name = "y1_location")
+    public int getY1Location() {
+        return y1Location;
+    }
+
+    public void setY1Location(int y1Location) {
+        this.y1Location = y1Location;
+    }
+
+    @Basic
+    @Column(name = "x2_location")
+    public int getX2Location() {
+        return x2Location;
+    }
+
+    public void setX2Location(int x2Location) {
+        this.x2Location = x2Location;
+    }
+
+    @Basic
+    @Column(name = "y2_location")
+    public int getY2Location() {
+        return y2Location;
+    }
+
+    public void setY2Location(int y2Location) {
+        this.y2Location = y2Location;
+    }
+
+    @Basic
+    @Column(name = "rotation")
+    public String getRotation() {

```

Comment:
String? jak bedziemy to przedstawiać?

Reply:
Poczytaj ten plik z SQL pls........................................... tylko po to go napisałem

Rotation = North/South/West/East